src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    protected ArrayList<Task> arrayList;
+    TaskList(ArrayList<Task> loaded) {
```

Comment:
```suggestion
    protected ArrayList<Task> arrayList;
    
    TaskList(ArrayList<Task> loaded) {
```
You may want to add a line break to improve readability.